feat: DIA-1360: Add token cost KPI to the Prompt aggregate-level subset metrics

niklub opened the pull request:
- https://github.com/HumanSignal/label-studio-sdk/pull/300 linked by [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/10750631127)

Commit:
feat: DIA-1360: Add token cost KPI to the Prompt aggregate-level subset metrics

Commit:
Add migrations

farioas commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10739156636)
> Successfully merged:  create mode 100644 web/libs/editor/src/components/Choice/Choice.module.scss

farioas committed:
Merge branch 'develop' into 'fb-dia-1360'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10739156636

farioas commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10739264397)

farioas committed:
Merge branch 'develop' into 'fb-dia-1360'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10739264397

farioas commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10739357798)
> Successfully merged:  1 file changed, 1 deletion(-)

farioas committed:
Merge branch 'develop' into 'fb-dia-1360'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10739357798

farioas commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10740356315)
> Successfully merged:  12 files changed, 19 insertions(+), 18 deletions(-)

farioas committed:
Merge branch 'develop' into 'fb-dia-1360'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10740356315

farioas commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10740790847)
> Successfully merged:  9 files changed, 9 insertions(+), 35 deletions(-)

Commit:
Add tests for PredictionMeta creation

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10750631127

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10750665291

Commit:
fmt

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10750970429

Commit:
Make create_no_commit methods in Prediction and PredictionMeta

Commit:
fmt

pakelley commented on label_studio/tasks/models.py:
```diff
@@ -997,6 +999,48 @@ def delete(self, *args, **kwargs):
+
+            # given the data receive, create annotation regions in LS format
+            # e.g. {"sentiment": "positive"} -> {"value": {"choices": ["positive"]}, "from_name": "", "to_name": "", ..}
+            pred = PredictionValue(result=label_interface.create_regions(data)).model_dump()
```
nit: I figure it's slightly better to just use the PredictionValue here, and we can just do `pred.result` below rather than `pred['result']`?
Obviously not a bit deal, but generally it seems like we've been quick to `model_dump` things, and I'd like to just use the pydantic model where we can.

pakelley reviewed (approved):
just one extremely minor thing, looks great

Commit:
Address comments

niklub commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10792867448)
> Successfully merged:  81 files changed, 1782 insertions(+), 439 deletions(-)

niklub committed:
Merge branch 'develop' into 'fb-dia-1360'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10792867448

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10792940623

robot-ci-heartex committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10793052041
